README.md

```diff
@@ -66,7 +66,9 @@ a number of programs to be installed:
 
 -   [Template Toolkit].  The Debian package is `libtemplate-perl`
 -   [Pandoc].  The Debian package is `pandoc`
--   `python3`, `wget` and `OpenSSL::Query` perl module.
+-   [OpenSSL::Query], a perl module that you install in normal perly way.
```

Comment:
"...install in normal perly way" may be a completely obvious (and hence superfluous) or a completely unhelpful description of how to do it, depending on whether you are a Perl Guru or not. 😉

Reply:
Agreed, I almost commented about this too.  A pointer to the readme perchance?

Reply:
Did you bother to follow the link?

Reply:
I did see the link.  I still think this could be improved.

Reply:
See the fixup commit I just pushed